src/hooks/usePropsGetters/usePreviousQuestionButtonProps.ts

```diff
@@ -0,0 +1,24 @@
+import { useCallback } from 'react';
+import { QuizAPIReducerState } from '../../components/CioQuiz/quizApiReducer';
+import { GetPreviousQuestionButtonProps, QuizEventsReturn } from '../../types';
+
+export default function usePreviousQuestionButtonProps(
+  quizApiState: QuizAPIReducerState,
+  previousQuestion: QuizEventsReturn.PreviousQuestion
+): GetPreviousQuestionButtonProps {
+  const getPreviousQuestionButtonProps: GetPreviousQuestionButtonProps = useCallback(() => {
+    const buttonDisabled = quizApiState.quizCurrentQuestion?.isFirstQuestion;
+    return {
+      title: 'Quiz Back Button',
+      role: 'button',
+      className: 'cio-question-back-button',
+      type: 'button',
+      onClick: () => previousQuestion(),
+      style: {
+        display: buttonDisabled ? 'none' : '',
```

Comment:
instead of limiting options for consumer of this lib, I'd prefer that we go with something like this:
```js
className={`cio-question-back-button ${buttonDisabled ? 'disabled' : ''}`}
```

Reply:
@mocca102 thoughts on the above?